sites/skeleton.dev/src/lib/modals/examples/ModalExampleForm.svelte

```diff
@@ -19,7 +19,7 @@
 
 	// We've created a custom submit function to pass the response and close the modal.
 	function onFormSubmit(): void {
-		if ($modalStore[0].response) $modalStore[0].response(formData);
+		if ($modalStore[0].response && $modalStore[0].type === 'component') $modalStore[0].response(formData);
```

Comment:
I'm afraid this may be the point where this change demonstrates that it's a breaking one for types.